src/main/java/bio/terra/service/dataset/exception/RetryQueryException.java

```diff
@@ -0,0 +1,23 @@
+package bio.terra.service.dataset.exception;
```

Comment:
Same as above. This seems like it is not dataset-specific. Could you move it in to the common package instead?